Symfony/src/AppBundle/Form/Type/RegistrationType.php

```diff
@@ -11,6 +11,9 @@
 
 class RegistrationType extends AbstractType
 {
+    /**
+     * @SuppressWarnings(PHPMD.UnusedLocalVariable)
```

Comment:
Oh yes my bad forgot to inform. The $options parameter in buildForm is not used, but if I remove it, a compilation error is given.

Compile Error: Declaration of AppBundle\Form\Type\RegistrationType::buildForm() must be compatible with Symfony\Component\Form\FormTypeInterface::buildForm(Symfony\Component\Form\FormBuilderInterface $builder, array $options)